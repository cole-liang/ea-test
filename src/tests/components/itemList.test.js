import React from "react";
import { shallow } from "enzyme";
import ItemList from "./../../components/itemList";

describe("ItemList component test", () => {
  const list = require("../../resourses/festivals.json");
  const wrapper = shallow(<ItemList list={list}></ItemList>);

  it("should render ItemList", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
