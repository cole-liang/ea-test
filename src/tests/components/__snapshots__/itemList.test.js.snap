// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemList component test should render ItemList 1`] = `
<ul>
  <div
    key="LOL-palooza"
  >
    LOL-palooza
  </div>
  <ItemList
    list={
      Array [
        Object {
          "name": "Winter Primates",
          "recordLabel": "",
        },
        Object {
          "name": "Frank Jupiter",
          "recordLabel": "Pacific Records",
        },
        Object {
          "name": "Jill Black",
          "recordLabel": "Fourth Woman Records",
        },
        Object {
          "name": "Werewolf Weekday",
          "recordLabel": "XS Recordings",
        },
      ]
    }
  />
  <div
    key="Small Night In"
  >
    Small Night In
  </div>
  <ItemList
    list={
      Array [
        Object {
          "name": "Squint-281",
          "recordLabel": "Outerscope",
        },
        Object {
          "name": "The Black Dashes",
          "recordLabel": "Fourth Woman Records",
        },
        Object {
          "name": "Green Mild Cold Capsicum",
          "recordLabel": "Marner Sis. Recording",
        },
        Object {
          "name": "Yanke East",
          "recordLabel": "MEDIOCRE Music",
        },
        Object {
          "name": "Wild Antelope",
          "recordLabel": "Marner Sis. Recording",
        },
      ]
    }
  />
  <div
    key="Trainerella"
  >
    Trainerella
  </div>
  <ItemList
    list={
      Array [
        Object {
          "name": "Wild Antelope",
          "recordLabel": "Still Bottom Records",
        },
        Object {
          "name": "Manish Ditch",
          "recordLabel": "ACR",
        },
        Object {
          "name": "Adrian Venti",
          "recordLabel": "Monocracy Records",
        },
        Object {
          "name": "YOUKRANE",
          "recordLabel": "Anti Records",
        },
      ]
    }
  />
  <div
    key="Twisted Tour"
  >
    Twisted Tour
  </div>
  <ItemList
    list={
      Array [
        Object {
          "name": "Summon",
          "recordLabel": "Outerscope",
        },
        Object {
          "name": "Auditones",
          "recordLabel": "Marner Sis. Recording",
        },
        Object {
          "name": "Squint-281",
        },
      ]
    }
  />
  <ItemList
    list={
      Array [
        Object {
          "name": "Critter Girls",
          "recordLabel": "ACR",
        },
        Object {
          "name": "Propeller",
          "recordLabel": "Pacific Records",
        },
      ]
    }
  />
</ul>
`;
